CRM/Core/BAO/FinancialTrxn.php

```diff
@@ -50,37 +50,38 @@ public function __construct() {
    * @param array $params
    *   (reference ) an assoc array of name/value pairs.
    *
-   * @param string $trxnEntityTable
-   *   Entity_table.
+   * @param array $ids
```

Comment:
On re-reading, it looks like $ids should just be removed as a parameter as you are altering the places that use it